testapp/admin.py

```diff
@@ -2,10 +2,15 @@
 
 from privates.admin import PrivateMediaMixin
 
-from .models import File
+from .models import File, File2
 
 
 @admin.register(File)
 class FileAdmin(PrivateMediaMixin, admin.ModelAdmin):
     private_media_fields = ("file", "image")
     private_media_view_options = {"attachment": True}
+
+
+@admin.register(File2)
+class FileAdmin2(FileAdmin):
+    no_download_fields = ("image",)
```

Comment:
you still need to properly mark the field as a private media field: `private_media_fields = ("image",)`